logdna/index.js

```diff
@@ -6,7 +6,7 @@ const zlib = require('zlib');
 
 // Constants
 const MAX_LINE_LENGTH = parseInt(process.env.LOGDNA_MAX_LINE_LENGTH) || 32000;
-const MAX_REQUEST_TIMEOUT = parseInt(process.env.LOGDNA_MAX_REQUEST_TIMEOUT) || 300;
+const MAX_REQUEST_TIMEOUT = parseInt(process.env.LOGDNA_MAX_REQUEST_TIMEOUT) || 30000;
```

Comment:
The explanation is a good case for renaming this var to `MAX_REQUEST_TIMEOUT_MS`.

Reply:
Nice catch! Done!